Automattic/wp-calypso
# Gutenboarding: reverting Edison screen shot for now to show the post image

## Changes proposed in this Pull Request

We last updated the Gutenboarding screenshots in https://github.com/Automattic/wp-calypso/pull/43561 

Little did we know that Edison was missing an image. 

<img width="1244" alt="Screen Shot 2020-07-08 at 10 12 48 am" src="https://user-images.githubusercontent.com/6458278/86858256-a6757680-c103-11ea-92d2-795ed3c613e0.png">


##  Testing instructions

Head over to `/new` on this branch and make sure the Edison design screenshot has the post image 👍
